### ISSUE TITLE
eventName might contain double escaped strings

### ISSUE THREAD

**Issue by lukepower** [pusher/pusher-http-dotnet#60]
Hello,

I have an issue in combination with a Laravel application, where I need to add the namespace to the event name. Something like this:
```
Task<ITriggerResult> resultTask = pusher.TriggerAsync("my-channel", @App\Events\my-event", "some data");
```
Will result in an event being fired with the event name being "App\\Events\\my-event". Somehow the escaped string gets translated in a string with double backslashes instead of single ones.

**Comment by erwin42** [comment]
How are you identifying the double backslashes?
Remember that backslashes are escaped in Json and in C# strings as \\. You can get easily deceived by the debugger or a Json debug message; for example:
my-event received: {"event":"PusherServerTest\\my-event","data":"{\"Message\":\"hello world\"}","channel":"my-channel"}

**Comment by erwin42** [comment]
Oh dear, my message above had the backslashes un-escaped. Let me try again
How are you identifying the double backslashes?
Remember that backslashes are escaped in Json and in C# strings as \\\\. You can get easily deceived by the debugger or a Json debug message; for example
my-event received: {"event":"PusherServerTest\\\\my-event","data":"{\\"Message\\":\\"hello world\\"}","channel":"my-channel"}
